python/ray/experimental/workflow/tests/test_virtual_actor.py

```diff
@@ -0,0 +1,46 @@
+import ray
+
+from ray.experimental import workflow
+
+
+@workflow.actor
+class Counter:
+    def __init__(self, x: int):
+        self.x = x
+
+    def get(self):
+        return self.x
+
+    def incr(self):
+        self.x += 1
+        return self.x
+
+    def __getstate__(self):
+        return self.x
+
+    def __setstate__(self, state):
+        self.x = state
+
+
+@workflow.step
+def init_virtual_actor(x):
+    return x
+
+
+def test_readonly_actor():
```

Comment:
Use fixture instead.

Reply:
`ray_start_regular` you can refer to s3 pr for a reference.

Reply:
I plan to update all tests using fixture in a later PR, so I can batch fix all tests.